zanata-war/src/main/resources/db/changelogs/db.changelog-1.5.xml

```diff
@@ -134,13 +134,24 @@
 	</changeSet>
 	
 	<changeSet author="aeng@redhat.com" id="8" dbms="mysql">
+        <validCheckSum>7:1d46e83838884b75859df2299905d8a0</validCheckSum>
         <comment>Alter glossary tables - remove sourceRef/add auto increment</comment>
         
         <dropColumn tableName="HGlossaryTerm" columnName="sourceRef" />
         
+        <dropUniqueConstraint tableName="HGlossaryTerm" constraintName="UKglossaryterm_glossary_entry_id"/>
+        <dropPrimaryKey tableName="HGlossaryEntry"/>
+        <dropPrimaryKey tableName="HGlossaryTerm"/>
+        <dropPrimaryKey tableName="HTermComment"/>
         <addAutoIncrement tableName="HGlossaryEntry" columnName="id" columnDataType="bigint"/>
         <addAutoIncrement tableName="HGlossaryTerm" columnName="id" columnDataType="bigint"/>
         <addAutoIncrement tableName="HTermComment" columnName="id" columnDataType="bigint"/>
+        <addPrimaryKey tableName="HGlossaryEntry" columnNames="id" />
+        <addPrimaryKey tableName="HGlossaryTerm" columnNames="id" />
+        <addPrimaryKey tableName="HTermComment" columnNames="id" />
+        <addForeignKeyConstraint baseTableName="HGlossaryTerm"
+                baseColumnNames="glossaryEntryId" constraintName="UKglossaryterm_glossary_entry_id"
+                referencedTableName="HGlossaryEntry" referencedColumnNames="id" />
```

Comment:
Can you add an xml comment with the reason for removing then adding the keys and constraints? I assume that the `addAutoIncrement` lines cannot be applied while the keys and constraints are in place, but we should not have to make such an assumption.